packages/files-ui/src/Components/FilesRoutes.tsx

```diff
@@ -43,7 +43,8 @@ export const ROUTE_LINKS = {
     const adjustedRawCurrentPath = !rawCurrentPath ? "/" : rawCurrentPath
     return `/shared/${bucketId}${adjustedRawCurrentPath}`
   },
-  DiscordInvite: "https://discord.gg/zAEY37fNb2"
+  DiscordInvite: "https://discord.gg/zAEY37fNb2",
+  SubscriptionWhitelistForm: ""
```

Comment:
Don't think this is needed.
```suggestion
  DiscordInvite: "https://discord.gg/zAEY37fNb2"
```